packages/shared/sdk-client/src/api/LDOptions.ts

```diff
@@ -80,6 +80,13 @@ export interface LDOptions {
    */
   capacity?: number;
 
+  /**
+   * Enables logging for debugging.
```

Comment:
```suggestion
   * Enables debug logging.
```

Or "Enables the debug log level."

Probably also needs "Has no effect when a custom logger is provided."